source/process/community-guidelines.rst

```diff
@@ -0,0 +1,141 @@
+ 
+ - create the issue with the mindset that the contributor might have no or limited experience with the Mattermost code base and limited exposure to the Mattermost product
+    
+- has an introductory section, with
+ 
+ - an invitation to join the `Contributors channel <https://pre-release.mattermost.com/core/channels/tickets>`_
+ - an invitation to join the `Developers channel <https://pre-release.mattermost.com/core/channels/developers>`_ for technical questions
+ - a link to the `developer’s guide <https://docs.mattermost.com/guides/developer.html>`_ for `machine setup <https://docs.mattermost.com/developer/developer-setup.html>`_ and `developer workflow <https://docs.mattermost.com/developer/developer-flow.html>`_
+ - a link to the `code contribution guidelines <https://docs.mattermost.com/developer/contribution-guide.html>`_
+    
+- has a well-defined difficulty level, both in the description and with the corresponding label
+ 
+ - Introductory: for someone new to the codebase. Guideline: 2-mana tickets are typically introductory
+ - Intermediate: for someone with a general knowledge of the codebase. Guideline: 4-mana tickets are typically introductory
+ - Advanced: for someone with a strong grasp of the codebase, e.g. implementing a major feature. Guideline: 8-mana tickets (or higher) are typically introductory     
+    
+- has a “Go” or “ReactJS” label
+ 
+ - helps contributors know what programming languages will be involved
+ - motivates contributors to work on specific tickets, e.g. someone learning Go might be motivated to tackle introductory Go tickets
+ - in general, if the work is all UI (client side), add a ReactJS label - if it's server side then add a Go label.
+
+- has a link to the corresponding JIRA ticket 
+
+- gets noted in the corresponding JIRA ticket
+ 
+ - ensures core devs are aware a help wanted issue was created, and to check whether someone in the community already has it in progress
+ - example text: ``GitHub Help Wanted issue here: [Link]. Before starting to work on this ticket, please check whether a community member has it in progress by checking the GitHub issue or by asking a PM.``
+
+Managing help wanted issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once a help wanted issue is added, the creator is responsible for
+
+- answering questions from community members, which get automatically posted in the `Community Heartbeat channel <https://pre-release.mattermost.com/core/channels/community-heartbeat>`_
+- following up with a community member if there hasn't been a response for two weeks
+- being the first tester once a pull request has been submitted
+
+Closing help wanted issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once a pull request for an open GitHub help wanted issue is merged, please ensure:
+
+- the corresponding GitHub help wanted issue is closed, thanking the contributor
+- the corresponding JIRA ticket is resolved
+
+----
+
+Community Forums
```

Comment:
This looks good, I'm wondering about the best way to fit some of this stuff together - I have the doc with the overall Community PM responsibilities which semi-overlaps with the response timeline portion, and IT has a communication doc that semi-overlaps with the response writing tips section.  

Not sure if we should somehow link between them, or if the duplication is fine for now.